OpenSim/Moco/Test/testMocoGoals.cpp

```diff
@@ -300,8 +300,23 @@ TEMPLATE_TEST_CASE("Test tracking goals", "", MocoCasADiSolver,
     SECTION ("MocoOrientationTrackingGoal") {
         MocoStudy studyOrientationTracking =
                 setupMocoStudyDoublePendulumMinimizeEffort<TestType>();
-        testDoublePendulumTracking<TestType, MocoOrientationTrackingGoal>(
+
+        SECTION("normal") {
+            testDoublePendulumTracking<TestType, MocoOrientationTrackingGoal>(
                 studyOrientationTracking, solutionEffort);
+        }
+
+        SECTION("removed column") {
```

Comment:
Consider a more descriptive section name, e.g., `"Missing coordinate value in states reference"`.